iris_grib/_save_rules.py

```diff
@@ -665,6 +665,16 @@ def set_fixed_surfaces(cube, grib):
         output_unit = cf_units.Unit('K')
         v_coord = cube.coord("air_potential_temperature")
 
+    elif cube.coords("level_height"):
+        grib_v_code = 118
+        output_unit = None
+        v_coord = cube.coord("level_height")
+
+    elif cube.coords("model_level_number"):
+        grib_v_code = 119
```

Comment:
i think that 119 is reasonably represented as `model_level_number`
GRIB code table 4.5 states
`119 Hybrid pressure level `

I think that this is interpreted as 'model levels' with extra coefficients stored elsewhere in teh message.  the ECMWF GRIB API has some functionality to support this iirc